examples/employee-app/src/main/java/com/example/helidon/employee/EmployeeMockList.java

```diff
@@ -0,0 +1,72 @@
+/* Copyright 2019 (c) Oracle and/or its affiliates. All rights reserved. */

```

Comment:
This should use the proper copyright header.

Reply:
This should be fixed now.